src/xrpld/app/tx/detail/CreateOffer.cpp

```diff
@@ -367,23 +396,23 @@ CreateOffer::bridged_cross(
     OfferStream offers_direct(
         view,
         view_cancel,
-        Book(taker.issue_in(), taker.issue_out()),
+        Book(taker.issue_in(), taker.issue_out(), std::nullopt),
```

Comment:
why is it std::nullopt here?

Reply:
this is a deprecated code that will never use domain. AMM doesn't support domain either.

Reply:
Got it, thanks 👍